docusaurus/docs/create-a-plugin/develop-a-plugin/build-a-panel-plugin.md

```diff
@@ -53,7 +53,7 @@ The [PanelProps](https://github.com/grafana/grafana/blob/747b546c260f9a448e2cb56
 
 You can access the panel properties through `props`, as seen in your plugin.
 
-**src/SimplePanel.tsx**
+**src/components/SimplePanel.tsx**
 
 ```js
```

Comment:
This looks significantly different in a newly scaffolded plugin?

`export const SimplePanel: React.FC<Props> = ({ options, data, width, height }) => {``